src/assets/iconComponents/Gmail.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+function SvgGmail() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 64 64"
+      width="15px"
```

Comment:
Always get a clean branch from a main, no need to include button implementation in dummy data

Reply:
Overlooked, my bad!